Owin.Security.Providers/Owin.Security.Providers.csproj

```diff
@@ -448,8 +464,16 @@
     <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="DoYouBuzz\" />
```

Comment:
Oh sorry, it is a future provider I prepare to provide. I haven't seen that I check-in